docs/pages/careers/react-tech-lead-core.md

```diff
@@ -18,7 +18,7 @@ We're a company of 31+ people as of late 2023, and we're growing.
 ## The products
 
 MUI is best known for our flagship product, Material UI—but this is just one of three core component libraries we maintain.
-Base UI is our headless component library, and Joy UI is a sister library to Material UI that implements our own in-house Joy Design system.
+Base UI is our headless component library, and Pigment CSS is a zero-runtime CSS-in-JS library, that is a replacement of the current styling engine using emotion/styled-components under the hood.
```

Comment:
```suggestion
Base UI is our headless component library, and Pigment CSS is a zero-runtime CSS-in-JS library, developed to be the next step from Emotion/styled-components.
```

The previous writing was sounding too Material UI-centric (it was also implying that it's a replacement of _Material's_ styling engine). I guess we can be just broader and say that it's the next step from Emotion/styled-components when it comes to CSS-in-JS? cc @samuelsycamore for additional thoughts/suggestions!